iib/workers/tasks/build.py

```diff
@@ -1004,6 +1021,10 @@ def handle_add_request(
         ``cnr_token`` or ``organization`` is not specified.
     """
     _cleanup()
+
+    if registry_auths:
+        create_dockerconfig(registry_auths)
```

Comment:
@lcarva looks like with dockerconfig added I was able to inspect image in registry.redhat.io in local dev environment  http://pastebin.test.redhat.com/948693